experimental/torchprep/torchprep/quantization.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+
+import torch
+
+from .format import Device, Precision
+from .utils import load_model
+
+
+def _quantize(
+    model_path: Path,
+    precision: Precision,
+    output_name: str = "quantized_model.pt",
+    device: Device = Device.cpu,
+) -> torch.nn.Module:
+
+    model = load_model(model_path, device)
+
+    if device == Device.cpu:
+        if precision == Precision.int8:
+            dtype = torch.qint8
+        elif precision == Precision.float16:
+            dtype = torch.float16
+        else:
+            print("unsupported {precision}")
+            return
+
+    quantized_model = torch.quantization.quantize_dynamic(
```

Comment:
It will be good to clarify in doc, that this would be most beneficial for cases where weights are the bottlenecks.

Reply:
All of the current optimizations are about the weights, will revisit other stuff like data loading and kernels in a future PR